.tool-versions

```diff
@@ -0,0 +1,3 @@
+ruby 2.7.5
+nodejs 12.16.1
+java zulu-8.56.0.21
```

Comment:
Should this file be in `.gitignore`?  Hyrax should work across multiple versions of tools.

Reply:
Opened a PR for this: https://github.com/samvera/hyrax/pull/5536